python/sglang/srt/model_executor/model_runner.py

```diff
@@ -375,10 +375,10 @@ def init_torch_distributed(self):
         # Check memory for tensor parallelism
         local_gpu_memory = get_available_gpu_memory(self.device, self.gpu_id)
         if self.tp_size > 1:
-            if min_per_gpu_memory < local_gpu_memory * 0.9:
```

Comment:
can we make it configurable, or is it possible to provide an option to disable it?

Reply:
add a env var `SGL_DISABLE_TP_MEMORY_INBALANCE_CHECK` to disable this check